src/main/java/seedu/address/model/person/MoneyOwed.java

```diff
@@ -14,6 +14,7 @@ public class MoneyOwed {
     public static final String VALIDATION_REGEX = "^(?:-)?\\d+(\\.\\d{0,2})?";
 
     public final Float moneyOwed;
+    public final boolean isNegative;
```

Comment:
Would it be better to turn this into a boolean function instead? So then when we update `moneyOwed` we don't have to update this attribute as well, less possibilities for bugs to occur.